collect_app/src/main/java/org/odk/collect/android/views/SuppressFlingGestureSlider.java

```diff
@@ -0,0 +1,42 @@
+package org.odk.collect.android.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.material.slider.Slider;
+
+import org.jetbrains.annotations.NotNull;
+
+public class SuppressFlingGestureSlider extends Slider implements  Slider.OnSliderTouchListener {
```

Comment:
I like the general pattern here. Do you think it'd make more sense for the value (and the class) to be named around `TrackingTouch...` rather than `SuppressFling...`?  The `Slider` doesn't actually suppress the gesture that's upstream from here.

Reply:
Oh okay yes right `Slider` is not suppressing the gesture here, it's just passing the value.